org.jung.ant-contrib.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "name": "org.doctales.ant-contrib",
+    "name": "org.jung.ant-contrib",
     "description": "Utility plugin which provides the ant-contrib library.",
     "keywords": ["Ant"],
     "homepage": "https://github.com/doctales/org.doctales.ant-contrib/",
```

Comment:
Might want to update this link while you're at it.

GitHub currently redirects to the new location, but that might not work forever.

Reply:
I'm sorry @infotexture. I've fixed it.

Reply:
Nice, even better than the GitHub URL. 🙂